src/main/java/com/box/sdk/BoxFileRequest.java

```diff
@@ -0,0 +1,434 @@
+                } else if (memberName.equals("created_at")) {
+                    this.createdAt = BoxDateFormat.parse(value.asString());
+                } else if (memberName.equals("created_by")) {
+                    JsonObject userJSON = value.asObject();
+                    String userID = userJSON.get("id").asString();
+                    BoxUser user = new BoxUser(getAPI(), userID);
+                    this.createdBy = user.new Info(userJSON);
+                } else if (memberName.equals("description")) {
+                    this.setDescription(value.asString());
+                } else if (memberName.equals("etag")) {
+                    this.etag = value.asString();
+                } else if (memberName.equals("expires_at")) {
+                    this.expiresAt = BoxDateFormat.parse(value.asString());
+                } else if (memberName.equals("folder")) {
+                    JsonObject folderJSON = value.asObject();
+                    String folderID = folderJSON.get("id").asString();
+                    BoxFolder folder = new BoxFolder(getAPI(), folderID);
+                    this.folder = folder.new Info(folderJSON);
+                } else if (memberName.equals("is_description_required")) {
+                    this.isDescriptionRequired = value.asBoolean();
+                } else if (memberName.equals("is_email_required")) {
+                    this.isEmailRequired = value.asBoolean();
+                } else if (memberName.equals("status")) {
+                    this.status = Status.fromJSONString(value.asString());
+                } else if (memberName.equals("title")) {
+                    this.title = value.asString();
+                } else if (memberName.equals("updated_at")) {
+                    this.updatedAt = BoxDateFormat.parse(value.asString());
+                } else if (memberName.equals("updated_by")) {
+                    JsonObject userJSON = value.asObject();
+                    String userID = userJSON.get("id").asString();
+                    BoxUser user = new BoxUser(getAPI(), userID);
+                    this.createdBy = user.new Info(userJSON);
+                } else if (memberName.equals("url")) {
+                    try {
+                        String urlString = value.asString();
+                        this.url = new URL(urlString);
+                    } catch (MalformedURLException e) {
+                        throw new BoxAPIException("Couldn't parse url for file request", e);
+                    }
+                }
+            } catch (Exception e) {
+                throw new BoxDeserializationException(memberName, value.toString(), e);
+            }
+        }
+    }
+
+    /**
+     * Enumerates the possible download states of a zip.
+     */
+    public enum Status {
+        /**
+         * Succeeded in downloading.
+         */
+        ACTIVE("active"),
+
+        /**
+         * Downloading in progress.
```

Comment:
should this mention something like in the docs - "An inactive file request will no longer accept new submissions, and any visitor to the file request URL will receive a HTTP 404 status code."